hive-event-listeners/README.md

```diff
@@ -8,7 +8,12 @@ Currently, the following modules are defined:
  - [apiary-metastore-auth](apiary-metastore-auth) - ReadOnlyAuth pre-event listener which will handle authorization using a configurable database whitelist.
  - [apiary-ranger-metastore-plugin](apiary-ranger-metastore-plugin) - RangerAuth pre-event listener which handles authorization and auditing using Ranger.
 
+### Note
+This **does not** include the following modules:
```

Comment:
I think instead of framing this as a negative ("does not") we should say something like:
"Related modules which also provide Hive Metastore Event Listeners for Kafka and SNS can be found at:"